src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -94,20 +95,21 @@ public class BlockStoreQueue : IBlockStoreQueue
             IChainState chainState,
             IBlockStoreQueueFlushCondition blockStoreQueueFlushCondition,
             StoreSettings storeSettings,
+            NodeSettings nodeSettings,
             IBlockRepository blockRepository,
             ILoggerFactory loggerFactory,
             INodeStats nodeStats)
         {
             Guard.NotNull(blockStoreQueueFlushCondition, nameof(blockStoreQueueFlushCondition));
             Guard.NotNull(chainIndexer, nameof(chainIndexer));
             Guard.NotNull(chainState, nameof(chainState));
-            Guard.NotNull(storeSettings, nameof(storeSettings));
+            Guard.NotNull(nodeSettings, nameof(nodeSettings));
```

Comment:
Is the removal of the `Guard` for `storeSettings` intentional?

Reply:
No, it's a mistake. 
First I replaced store settings with node settings, then saw that I still need node settings for some value and readded it while forgetting to reintroduce the guard. 
Also it seems some of the other params lack guards.

Fixed.